### ISSUE TITLE
Instruments method fails with no method "upper"

### ISSUE THREAD

**Issue by toymachiner62** [aurbano/robinhood-node#14]
With the following code I get the error:

```sh
TypeError: Object GOOG has no method 'upper'
    at Object.RobinhoodWebApi.api.instruments (/Users/tomcaflisch/Sites/robinhood-riches/node_modules/robinhood/src/robinhood.js:116:29)
...
```

```js
const Robinhood = require('robinhood');
const credentials = require('./credentials');

const trader = Robinhood(credentials, function() {
  trader.quote_data('NYMX', function(err, httpResponse, body) {
    if(err) {
      console.error(err);
      return;
    }
    console.log('Quote data = ', body);
  });

  trader.instruments('NYMX', function(err, httpResponse, body) {
    console.log('err = ', err);
    console.log('instruments = ', body);
  });
});
```

**Comment by nodesocket** [comment]
Seems like the `ticker` parameter in `quote_data()` is case-sensitive which it should not be.

```
➜  node quotes --ticker AAPL
{ results:
   [ { ask_price: '96.9100',
       ask_size: 4100,
       bid_price: '96.9000',
       bid_size: 400,
       last_trade_price: '96.9100',
       last_extended_hours_trade_price: '96.8900',
       previous_close: '96.7600',
       adjusted_previous_close: '96.7600',
       previous_close_date: '2016-02-25',
       symbol: 'AAPL',
       trading_halted: false,
       updated_at: '2016-02-26T21:00:00Z' } ] } 200
➜  node quotes --ticker aapl
undefined 400
```

Anybody else think we should run `.toUpper()` anywhere in this package we are passing in a symbol?

**Comment by aurbano** [comment]
This is fixed and will be released today as v0.3.0
